CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 #Changelog
 
+## 1.0.22-24
+### Bug Fixed
+- Updated log4j library versions
```

Comment:
Might be useful to specify the version we upgraded to

Reply:
done